frontend/views/pages/Home.vue

```diff
@@ -50,6 +50,19 @@ main.c-splash(data-test='homeLogo' v-if='!currentGroupId')
         @click='openModal("GroupJoinModal")'
         data-test='joinGroup'
       ) Join a Group
+
+  banner-simple.hide-hoverable-device.hide-tablet.c-pwa-promo(
```

Comment:
`.hide-hoverable-device.hide-tablet` ensures that this banner won't show in desktop.